api/src/main/java/org/openmrs/notification/mail/velocity/VelocityMessagePreparator.java

```diff
@@ -40,7 +41,11 @@ public class VelocityMessagePreparator implements MessagePreparator {
 	public VelocityMessagePreparator() throws MessageException {
 		try {
 			engine = new VelocityEngine();
-			engine.init();
+			Properties props = new Properties();
+			props.put("runtime.log.logsystem.class", "org.apache.velocity.runtime.log.SimpleLog4JLogSystem");
```

Comment:
It does look like Velocity recommends the `Log4JLogChute`class over this one.

Reply:
Changed.  Want to have a look?